elasticdl/python/tests/test_utils.py

```diff
@@ -275,6 +275,10 @@ def create_pserver(
         pserver = ParameterServer(args)
         pserver.prepare()
         pservers.append(pserver)
+
+    for channel in channels:
+        grpc.channel_ready_future(channel).result()
```

Comment:
We can try futures.wait(future_list)

Reply:
There is no `wait` in `_ChannelReadyFuture` of GRPC.
![image](https://user-images.githubusercontent.com/18071380/89750027-c9c58400-dafc-11ea-827a-24cfa5be26e8.png)

And we cannot use `_ChannelReadyFuture`  in ``futures.wait()` for the error:
![image](https://user-images.githubusercontent.com/18071380/89750060-eeb9f700-dafc-11ea-9baa-f924c4c26efc.png)